postman/README.md

```diff
@@ -0,0 +1,37 @@
+# Postman code examples
+
+This directory contains examples for use with Postman
+
+Script | Link |
```

Comment:
Is it valid markdown to drop the opening `|`?

Reply:
Added the opening |